app/javascript/dashboard/assets/scss/plugins/_dropdown.scss

```diff
@@ -3,7 +3,7 @@
   @include border-light;
   box-sizing: content-box;
   width: fit-content;
-  z-index: 999;
+  z-index: 9999;
```

Comment:
'app/javascript/shared/assets/stylesheets/z-index.scss' this has some defined variables. Can you try if it's possible with this ?

Reply:
Replaced the hardcoded value with a css var